src/routes/safe/store/actions/processTransaction.js

```diff
@@ -79,6 +84,23 @@ const processTransaction = ({
   }
 
   try {
+    if (!isExecution && !isSmartContractWallet && semverSatisfies(safeVersion, '>=1.1.1')) {
+      const signature = await tryOffchainSigning({ ...txArgs, safeAddress })
```

Comment:
Side comment: it feels super weird to see the same code added in multiple places

Reply:
We have two different actions for new transactions (when user sends a transaction) and processing transactions (approving/executing existing transaction from transaction list). There are definitely some code duplication but there's also a difference. If we merge this to one function IMO it will became big and harder to debug/maintain. Maybe in the future we can make an unified handler which would include the duplicated code and then call either of those functions, but I wouldn't do anything related to that now

Reply:
Yes I am aware, that is why it was more a note on the side, not something I would tackle right now ;)